tests/integration/minibroker_test.go

```diff
@@ -139,7 +139,54 @@ var _ = Describe("classes", func() {
 			name:   "mysql",
 			plan:   "5-7-30",
 			params: map[string]interface{}{},
-			assert: func(instance *apiv1beta1.ServiceInstance, binding *apiv1beta1.ServiceBinding) {},
+			assert: func(instance *apiv1beta1.ServiceInstance, binding *apiv1beta1.ServiceBinding) {
+				By("rendering and loading the mysql client template")
+				tmplPath := path.Join(testDir, "resources", "mysql_client.tmpl.yaml")
+				values := map[string]interface{}{
+					"DatabaseVersion": "5.7.30",
+					"SecretName":      binding.Spec.SecretName,
+					"Command": []string{
+						"sh", "-c",
+						"mysql" +
+							" --host=${DATABASE_HOST}" +
+							" --port=${DATABASE_PORT}" +
+							" --user=${DATABASE_USER}" +
+							" --password=${DATABASE_PASSWORD}" +
+							" --execute='SELECT 1'",
+					},
+				}
+				obj, err := testutil.LoadKubeSpec(tmplPath, values)
+				Expect(err).NotTo(HaveOccurred())
+
+				By("creating the mariadb client resource")
```

Comment:
Copy-pasta left-over.

```suggestion
				By("creating the mysql client resource")
```